### PR TITLE
Folders/aj 146 create folder required fields

### PR DESCRIPTION
**Pull request by Raymundogs**

### CODE REVIEW

File: src/main/java/clickup/FoldersRequests.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package clickup;
```
Comment (GustavoZHA) [inline src/main/java/clickup/FoldersRequests.java]: Add copyright

File: src/main/java/clickup/FoldersRequests.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import core.api.ApiMethod;
+import core.api.ApiRequest;
+import core.api.ApiRequestBuilder;
+import core.api.ApiResponse;
+
+public final class FoldersRequests {
+    private FoldersRequests() {
+    }
+
+    public static ApiResponse getFolder(final String folderId) {
```
Comment (GustavoZHA) [inline src/main/java/clickup/FoldersRequests.java]: Add javadoc

**Review by JorgeRCV — APPROVED** [review]

File: src/main/java/clickup/FoldersRequests.java (hunk 1 above)
Comment (Raymundogs) [reply src/main/java/clickup/FoldersRequests.java]: I realized, I was not using the class so I deleted it. Thanks.

File: src/main/java/clickup/FoldersRequests.java (hunk 2 above)
Comment (Raymundogs) [reply src/main/java/clickup/FoldersRequests.java]: I realized, I was not using the class so I deleted it. Thanks.

**Review by GustavoZHA — APPROVED** [review]

**Pull request merged**